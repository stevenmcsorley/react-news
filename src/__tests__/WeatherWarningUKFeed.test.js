import React from 'react';
import { shallow } from 'enzyme';
import Weather from '../components/news/WeatherWarningUKFeed'

test("testing Weather feed",  async () =>{

    const WeatherComp = shallow(<Weather />, 'div');
    expect(WeatherComp).toMatchSnapshot();

})