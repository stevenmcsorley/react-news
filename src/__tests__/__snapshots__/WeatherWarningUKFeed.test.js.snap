// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Weather feed 1`] = `ShallowWrapper {}`;
